Makefile

```diff
@@ -969,7 +969,7 @@ TEST_WEB_83 := \
 	test_web_custom \
 	test_web_zend_1_21
 
-FILTER := .
+FILTER := $(if $(FILTER),$(FILTER),".")
```

Comment:
This allows me to run a single test this way:
```
TESTS="tests/Integration/TracerTest.php" FILTER="testResourceNormalizationWebDefault" make test_integration
```
I don't know if there is a better way 🤷🏼‍♂️

Reply:
Even better. You'll be happy to know that you beat _copilot_ 😉